packages/backend/types/template.d.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+declare module '*.mustache?raw' {
```

Comment:
Can we move this declaration in types at the root directory ? https://github.com/projectatomic/ai-studio/tree/main/types

Reply:
I am following the pattern that has been initiated in Podman Desktop (kind, compose)

Reply:
When you merged https://github.com/projectatomic/ai-studio/pull/565/files you added the markdown module in the types folder, why would it be different here ?

Reply:
I did not get your point; should be good now